EvalView/templates/EvalView/pairwise-assessment-document-newui-esa.html

```diff
@@ -0,0 +1,841 @@
+{% extends "Dashboard/base.html" %}
```

Comment:
Please consider changing the name of this file to the shorter name proposed above: `pairwise-assessment-document-esa`.

Reply:
Changed.